packages/docprovider-extension/src/filebrowser.ts

```diff
@@ -50,21 +56,33 @@ const DOCUMENT_TIMELINE_URL = 'api/collaboration/timeline';
 /**
  * The default collaborative drive provider.
  */
-export const drive: JupyterFrontEndPlugin<ICollaborativeDrive> = {
+export const drive: ServiceManagerPlugin<YDrive> = {
   id: '@jupyter/docprovider-extension:drive',
+  autoStart: true,
   description: 'The default collaborative drive provider',
-  provides: ICollaborativeDrive,
-  requires: [ITranslator],
-  optional: [IGlobalAwareness],
+  provides: IDefaultDrive,
+  requires: [IUserManager],
+  activate: (_: null, user: User.IManager): YDrive => {
+    const drive = new YDrive(user);
+    return drive;
+  }
+};
+
+export const defaults: JupyterFrontEndPlugin<void> = {
+  id: '@jupyter/docprovider-extension:defaults',
+  autoStart: true,
+  requires: [IDefaultDrive],
+  optional: [IGlobalAwareness, ITranslator],
   activate: (
     app: JupyterFrontEnd,
-    translator: ITranslator,
-    globalAwareness: Awareness | null
-  ): ICollaborativeDrive => {
-    const trans = translator.load('jupyter_collaboration');
-    const drive = new YDrive(app.serviceManager.user, trans, globalAwareness);
-    app.serviceManager.contents.addDrive(drive);
-    return drive;
+    ydrive: YDrive,
+    globalAwareness: Awareness | null,
+    translator: ITranslator | null
+  ): void => {
+    ydrive.translator = (translator ?? nullTranslator).load('jupyterlab');
+    if (globalAwareness) {
+      ydrive.globalAwareness = globalAwareness;
+    }
```

Comment:
Because the service manager plugins are activated first and their `app` is still `null` during the first phase, we extract setting the `translator` and `globalAwareness` to a separate plugin, so it is activated as a regular plugin.